plugins/rtmp-services/data/services.json

```diff
@@ -1104,6 +1104,19 @@
                 "max video bitrate": 20000,
                 "max audio bitrate": 192
             }
+        },
+        {
+            "name": "Lahzenegar - لحظه نگار",
```

Comment:
It is missing the following line: "common": false,

Reply:
It seems you did not see my previous message on the other pull request:  the value is false by default.  You do not need to use it unless you're setting it to true.

Reply:
Jep, missed that one. I have seen some use it earlier, so expected that these also needed it. Check line 1078 for that.